services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/RateDetailsSummarySection.tsx

```diff
@@ -154,15 +138,15 @@ export const RateDetailsSummarySection = ({
                 {submission.rateInfos.map((rateInfo) => {
                     return (
                         <React.Fragment
-                            key={`${rateName(rateInfo)}${JSON.stringify(
-                                rateInfo.rateDocuments
-                            )}`}
+                            key={`${
+                                rateInfo.rateCertificationName
+                            }${JSON.stringify(rateInfo.rateDocuments)}`}
                         >
```

Comment:
This maybe an issue, but using these two pieces of data may not guarantee a unique key? I might opted here to add an `index` to the mix and as a part of my ticket [MR-2714](https://qmacbis.atlassian.net/browse/MR-2714) I could switch this to use the `id` in each rate info.

Reply:
ID would be great; are you saying you could roll that into your PR?  And while I don't think this guarantees a unique key, I think it's also very unlikely that we'd get a dupe on the same page.